yb-voyager/cmd/importDataFileTaskImporter.go

```diff
@@ -203,7 +205,13 @@ func (fti *FileTaskImporter) importBatch(batch *Batch) {
 	log.Infof("%q => %d rows affected", batch.FilePath, rowsAffected)
 	if err != nil {
 		if fti.errorHandler.ShouldAbort() {
-			utils.ErrExit("import batch: %q into %s: %s", batch.FilePath, batch.TableNameTup, err)
+			// TODO: standardize the return type of ImportBatch to return an ImportBatchError
+			var ibe errs.ImportBatchError
+			if errors.As(err, &ibe) {
+				// If the error is an ImportBatchError, we abort directly
+				utils.ErrExit("%w", err)
+			}
```

Comment:
if this will be the way for ImportBatchErrors, then we can move it to ErrExit function?

Reply:
This is only required temporarily, to avoid duplicating the "import batch: batch into table..." in the error message (that is part of line 214)

The plan is to make `tdb.ImportBatch` return an `ImportBatchError` struct instead of error interface; Once we do that, we won't have to have any special handling